### ISSUE TITLE
Qlog won't connect to FT857+RigExpert Standart through Omnirig

### ISSUE THREAD

**Issue by Javier-EA6ALW** [foldynl/QLog#473]
Hi folks,

I am trying to connect my radio through Omnirig to Qlog but it doesn't work at all. However, radio is readed on all other softwares I have installed on my PC such as JTDX, Log4OM and so.

Configuration in Qlog seems quite simple and fast CAT is not working for it.

TNX in advance for your time.

Cheers!

**Comment by Javier-EA6ALW** [comment]
Already found it! Newbie mistake on not checking on "connect to radio"

73!

**Comment by Javier-EA6ALW** [comment]
Okay, although radio is connected, Qlog doesn't show frequency (stays at 0.00 Hz), but I'm able to switch mode (USB, LSB, CW....)

73!

**Comment by foldynl** [comment]
Is the frequency option checked in the Rig settings?

**Comment by Javier-EA6ALW** [comment]
Yes, it actually is. I found that setting the radio through hamlib works perfect (VFO, frequency, band, PTT and so). So maybe there's an issue on how Qlog reads OmniRig information with this radio.

Using hamlib prevents me of using JTDX, MSHV or other software simultanously.

(hope I make myself understand, English is not my first language and always try not to use translator)

TNX!
73

**Comment by foldynl** [comment]
Do you use the omnirig v1 or v2 ?

**Comment by Javier-EA6ALW** [comment]
> Do you use the omnirig v1 or v2 ?
v1

**Comment by aa5sh** [comment]
Can you send a screenshot of your radio settings for OmniRig and in QLog?

**Comment by Javier-EA6ALW** [comment]
> Can you send a screenshot of your radio settings for OmniRig and in QLog?

![image](https://github.com/user-attachments/assets/06b20d92-d9cf-4742-92a2-8e38fe1d4496)

**Comment by Javier-EA6ALW** [comment]
> Can you send a screenshot of your radio settings for OmniRig and in QLog?

![image](https://github.com/user-attachments/assets/49de6010-f5e5-4b64-b445-50c0c6c46c34)

**Comment by Javier-EA6ALW** [comment]
> Ok I just tested on a Windows machine with Smart SDR and it seemed to work ok with either OmniRig v1 or v2. Only issue I had was at first both were configured to use the same COM port and were fighting. Do you other applications work ok with Omnirig and your radio and all? I'd still be curious to see your setup for both settings in Qlog Hamlib and Omnirig as well as the Omnirig settings them self.

Sorry for the late response. Sure, I send you some pictures now

![image](https://github.com/user-attachments/assets/11d37193-7514-4711-b9b2-452c5127c109)

![image](https://github.com/user-attachments/assets/f7f35069-c964-4660-bb30-af77576d9364)

I hope everything is clear although it is in Spanish. TNX for your help

**Comment by aa5sh** [comment]
So is OmniRig working with other applications?  Looks like there are some settings different between HamLib and Omnirig (Stop Bits) and Flow Control but that (Off versus High).

**Comment by Javier-EA6ALW** [comment]
> So is OmniRig working with other applications? Looks like there are some settings different between HamLib and Omnirig (Stop Bits) and Flow Control but that (Off versus High).

Yes, it worked with everything. I tried to change stop-bits to the same number but nothing happened. I've been using LOG4OM and also used HDR back in the day.
